src/tmx/Asn_J2735/include/asn_j2735_r63/TestMessage00.h

```diff
@@ -47,8 +47,8 @@ extern asn_TYPE_member_t asn_MBR_TestMessage00_1[2];
 #endif
 
 /* Referred external types */
-#include "Header.h"
-#include "RegionalExtension.h"
+// #include "Header.h"
```

Comment:
If these are no longer being used. Lets remove them

Reply:
The same answer below